libcontainerd/client_windows.go

```diff
@@ -299,7 +299,7 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
 		Owner:         defaultOwner,
 		TerminateOnLastHandleClosed: true,
 		HvRuntime: &hcsshim.HvRuntime{
-			ImagePath:       `c:\program files\lcow`,
+			ImagePath:       `c:\Program Files\Linux Containers`,
```

Comment:
Why does this use a hard-coded `c:\Program Files` instead of the `ProgramFiles` environment variable? Also, why not export this name from OpenGCS?

Nit: With changing to mixed case, I would capitalize the drive letter too.